opts/opts.go

```diff
@@ -277,7 +283,7 @@ func ValidateExtraHost(val string) (string, error) {
 }
 
 func ValidateLabel(val string) (string, error) {
-	if strings.Count(val, "=") != 1 {
+	if strings.Count(val, "=") < 1 {
 		return "", fmt.Errorf("bad attribute format: %s", val)
 	}
```

Comment:
@thaJeztah I don't think it was added :stuck_out_tongue:. I should have said "the expression is not the same" ; meaning it's not gonna be the same regexp to use anyway :smile: